### DISCUSSION TITLE
Tier Pricing

### DISCUSSION THREAD

**Discussion by osseonews** [medusajs/medusa#1326] [Q&A] [answered]
Does medusa offer tier price for specific skus? For example, if you buy 1 of ABC the price is $10 per item, but if you buy 5 of ABC the price drops to $5 per item.

**Comment by srindom** [comment, marked as answer]
We have recently added this with the introduction of Price Lists! 

Basically, you can set prices like this:
```
{ 
  variant_id: "variant_123",
  min_quantity: 1,
  max_quantity: 4,
  amount: 1000,
  currency_code: "usd",
},
{ 
  variant_id: "variant_123",
  min_quantity: 5,
  amount: 1000,
  currency_code: "usd",
}
```

We have not added this to Admin yet so you can currently only control it from the API.

**Reply by ddanninger** [reply]
Is it safe to use in 2024, or would it be better to look into some other way of implementing this in case this would be not supported anymore in the near future.

**Reply by kimskogsmo** [reply]
Will there ever be an admin UI for this? Considering Medusa for a new semi-large project but my client needs tiered prices per individual products (printed products) and an UI is definitely needed to manage all this, since he may have a large amount of products that need tiered prices, manually entering these pricing rules as a dev is not feasible.

**Comment by SGFGOV** [comment]
This is cool, but how do I use it without admin Integration.   Will I need
to use curl or something like that? Or does the medusa cli have some tool
that can issue the commands?



Best Regards
Govind

On Fri, 20 May 2022, 14:12 Sebastian Rindom, ***@***.***>
wrote:

> We have recently added this with the introduction of Price Lists!
>
> Basically, you can set prices like this:
>
> {
>   variant_id: "variant_123",
>   min_quantity: 1,
>   max_quantity: 4,
>   amount: 1000,
>   currency_code: "usd",
> },
> {
>   variant_id: "variant_123",
>   min_quantity: 5,
>   amount: 1000,
>   currency_code: "usd",
> }
>
> We have not added this to Admin yet so you can currently only control it
> from the API.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/medusajs/medusa/discussions/1326#discussioncomment-2789419>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AXEQJHHFJGAD4IGLPAVAKTTVK5F5PANCNFSM5TGWCKMQ>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>

-- 

This message (including any attachments) may contain confidential, 
proprietary, privileged and/or private
information. The information is 
intended to be for the use of the individual or entity designated above. If
you are not the intended recipient of this message, please notify the 
sender immediately, and delete the
message and any attachments. Any 
disclosure, reproduction, distribution or other use of this message or
any 
attachments by an individual or entity other than the intended recipient is 
prohibited.

**Comment by sashahilton00** [comment]
Is there any update on this being added to the UI? We seem to be in a state of limbo to a certain extent with medusa between v1 and v2 - there is clearly a v2.0 coming, but it's unclear when, in the meantime features such as this one which are pretty integral to any type of B2B offering are left unanswered/unimplemented in v1.0.

Could one of the devs jump in with an update? If the answer is 'probably better to wait for v2.0' that's absolutely fine, as much as anything it would be good to know if theres a plan to support this in the v1.0 UI, or if all focus is on v2, and we should wait until that releases. At any rate some guidance would be appreciated.

**Comment by gyurka00** [comment]
Is it possible with v0.2.6?

**Reply by sashahilton00** [reply]
I ended up writing a plugin for v1 - haven't updated it for v2, but might be worth a look, see if it's what you're looking for and how much work it is to make it work with v2 if any. Happy to accept PRs.

https://github.com/sashahilton00/medusa-products-pricing-groups

**Comment by atif** [comment]
@srindom Are there any plans to expose it in Admin in V2?

**Comment by alexvieru1** [comment]
Can anyone share or help me out with a working workflow for the tier pricing within v2? I've been struggling for a couple days and I don't seem to get anywhere.

**Comment by WoodyChau** [comment]
I've noticed that dealing with tier pricing through the API can be a bit of a hassle, especially since it doesn't have native support in the Admin UI. To make life easier, I’ve put together an open-source UI tool that makes managing tier pricing for Medusa projects a breeze. You can effortlessly set up and tweak pricing by creating templates that you can slap onto any product or variant you want, all without having to mess around with code.

If anyone's got fresh ideas or feedback, Would love to chat about ways to make this tool even better.

<img width="1558" height="1329" alt="Screenshot_1" src="https://github.com/user-attachments/assets/e33b0467-d293-4e77-8c12-001fb7f7288a" />
<img width="1556" height="1219" alt="Screenshot_2" src="https://github.com/user-attachments/assets/214a2d67-9b81-4269-83ea-e44c715f2f19" />
<img width="1270" height="713" alt="Screenshot_3" src="https://github.com/user-attachments/assets/ff62e57c-3006-4fcc-9597-42393a6b8dbe" />

Repo:
https://github.com/WoodyChau/medusajs-tier-price-bulk-control
